libraries/AP_Arming/AP_Arming.cpp

```diff
@@ -71,11 +71,12 @@
 #define AP_ARMING_BOARD_VOLTAGE_MAX     5.8f
 #define AP_ARMING_ACCEL_ERROR_THRESHOLD 0.75f
 #define AP_ARMING_AHRS_GPS_ERROR_MAX    10      // accept up to 10m difference between AHRS and GPS
+#define AP_ARMING_MAX_DIST 500
```

Comment:
this must be 0 to avoid behavior changes to existing setups...

Reply:
The `ARMING_MIS_ITEMS` bitmask disables the check by default, so the behaviour won't change

Reply:
thats fine